neuromation/cli/job.py

```diff
@@ -954,13 +955,21 @@ def format_fail(job: str, reason: Exception) -> str:
         available = await root.client.users.get_acl(
             root.client.username, scheme="storage"
         )
+        to_skip = set()
```

Comment:
Would not be simpler to collect a list of valid uris?
```python
storage_uris = []
for perm in available:
    ...
    storage_uris.append(perm.uri)
```

Reply:
Done